src/main/java/com/gradle/enterprise/bamboo/config/PluginEnabledListener.java

```diff
@@ -0,0 +1,47 @@
+package com.gradle.enterprise.bamboo.config;
+
+import com.atlassian.bamboo.bandana.PlanAwareBandanaContext;
+import com.atlassian.bandana.BandanaManager;
+import com.atlassian.event.api.EventListener;
+import com.atlassian.plugin.event.events.PluginEnabledEvent;
+import com.fasterxml.jackson.core.JsonProcessingException;
+import org.slf4j.Logger;
+import org.slf4j.LoggerFactory;
+import org.springframework.beans.factory.annotation.Autowired;
+import org.springframework.stereotype.Component;
+
+@Component
+public class PluginEnabledListener {
+
+    private static final Logger LOGGER = LoggerFactory.getLogger(PluginEnabledListener.class);
+    private static final String PLUGIN_KEY = "com.gradle.enterprise.gradle-enterprise-bamboo-plugin";
```

Comment:
We can be more specific with this variable, so it's more clear that an event listener gets all events but filters only for the specific plugin.   

```suggestion
    private static final String GRADLE_ENTERPRISE_BAMBOO_PLUGIN_KEY = "com.gradle.enterprise.gradle-enterprise-bamboo-plugin";
```